app/index.js

```diff
@@ -1025,11 +1025,9 @@ module.exports = JhipsterGenerator.extend({
             if (this.useSass) {
                 this.template(WEBAPP_DIR + 'scss/main.scss', WEBAPP_DIR + 'scss/main.scss');
                 this.template(WEBAPP_DIR + 'scss/vendor.scss', WEBAPP_DIR + 'scss/vendor.scss');
+            } else {
+                this.template(WEBAPP_DIR + 'content/css/main.css', WEBAPP_DIR + 'content/css/main.css');
             }
-            // this css file will be overwritten by the sass generated css if sass is enabled
```

Comment:
I made sure that sass, if enabled, is always run after `yo jhipster`, so this placeholder is no longer needed.